WalletWasabi/Hwi/Models/HardwareWalletModels.cs

```diff
@@ -9,19 +9,65 @@ namespace WalletWasabi.Hwi.Models
 	/// </summary>
 	public enum HardwareWalletModels
 	{
+		[Description("Hardware Wallet")]
 		Unknown,
+
+		[Description("Coldcard")]
 		Coldcard,
+
+		[Description("Coldcard Simulator")]
 		Coldcard_Simulator,
+
+		[Description("BitBox")]
 		DigitalBitBox_01,
+
+		[Description("BitBox Simulator")]
 		DigitalBitBox_01_Simulator,
+
+		[Description("KeepKey")]
 		KeepKey,
+
+		[Description("KeepKey Simulator")]
 		KeepKey_Simulator,
+
+		[Description("Ledger Nano S")]
 		Ledger_Nano_S,
+
+		[Description("Trezor One")]
 		Trezor_1,
+
+		[Description("Trezor One Simulator")]
 		Trezor_1_Simulator,
+
+		[Description("Trezor T")]
 		Trezor_T,
+
+		[Description("Trezor T Simulator")]
 		Trezor_T_Simulator,
+
+		[Description("BitBox")]
 		BitBox02_BTCOnly,
+
+		[Description("BitBox")]
 		BitBox02_Multi,
 	}
+
+	public static class EnumExtensions
```

Comment:
Can you create a new file with this under WalletWasabi/Extensions?

Reply:
Done. And shouldn't we refactor the `DequeueReason.cs` class? Because the situation is similar to here.